### PR TITLE
Daniel - AboutUs and PPP page

### PR DESCRIPTION
**Pull request by dandaandaaaaaan**
This PR adds my photo, description and template PPP page

Linked Issues: #55, #53

### CODE REVIEW

File: docs/team/daniel.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    * Credits:  `to be added soon`
+
+* **Code contributed**: `to be added soon`
+
+* **Documentation**:
+    * User Guide:
+        * Added documentation for the various `import` and `export` functions
+    * Developer's Guide:
+        * `to be added soon`
+* **Community**: `to be added soon`
```
Comment (sharpstorm) [inline docs/team/daniel.md:26]: I would like to request the template retaining more of the original sections. See my PR #66.

**Review by sharpstorm — CHANGES_REQUESTED** [review]
Please fix the following points that do not conform to the project specifications:
- The file name of the profile photo should be `docs/images/github_username_in_lower_case.png`.
- The file name of the PPP markdown file should be  `docs/team/github_username_in_lower_case.md`.

File: docs/team/daniel.md (hunk 1 above)
Comment (dandaandaaaaaan) [reply docs/team/daniel.md]: okay will follow thanks!

**Review by Wei-Xinn — APPROVED** [review]
lgtm!

**Review by sharpstorm — APPROVED** [review]
Thanks for changes, lgtm.

**Comment by sharpstorm** [comment]
Closes issue #55.

**Pull request merged**